modules/auxiliary/scanner/rdp/rdp_scanner.rb

```diff
@@ -42,7 +42,7 @@ def rdp?
     sock.put(@probe)
     response = sock.get_once(-1)
     if response
-      if RDP_RE.match?(response)
+      if RDP_RE.match(response)
```

Comment:
Ruby 2.2 is still supported. Stumbled on this error on Kali. We recently had a problem with `Numberic positve?` dealing with the same problem: https://github.com/rapid7/metasploit-framework/pull/8792